src/server/helpers/config.js

```diff
@@ -1,51 +1,42 @@
-import path from 'path';
-import nconf from 'nconf';
+import { Map } from 'immutable';
+import { argv } from 'yargs';
 import dotenv from 'dotenv';
```

Comment:
We only need to use `dotenv` in `src/server`. We won't ever need to use it in `src/common`. If we kept `nconf`, we would need to re-implement the `get`, `set` and `stores` features in universal code, because these are the features we currently depend on.

I think it's worth having a config API that is mirrored in implementation in server-only and universal code.